guide/C/ch_loans.xml

```diff
@@ -336,7 +336,7 @@ Continuous Interest: <emphasis>i = ln[(1+ieff)^PF]</emphasis></screen>
          X = 1 for beginning of period payments</screen>
         <para>From this equation, functions which solve for the individual variables can be derived. For a
           detailed explanation of the derivation of this equation, see the comments in the file
-          src/calculation/fin.c from the &app; source code. The A, B, and C variables are defined
+          <ulink url="https://github.com/Gnucash/gnucash/blob/maint/libgnucash/app-utils/calculation/fin.c"><filename>&file-finc;</filename></ulink> from the &app; source code. The A, B, and C variables are defined
```

Comment:
Why  not `<ulink url="&url-gh-gc-maint;&file-finc;"><filename>&file-finc;</filename></ulink>`?

Reply:
I misunderstood the definition of CDATA in attribute. According to the [XML definition](https://www.w3.org/TR/REC-xml/#attdecls), ENTITY is expanded within the attribute of CDATA.
So what you are saying is correct.

Reply:
7a15cb1